roles/static_inventory/templates/inventory.j2

```diff
@@ -57,6 +57,7 @@ etcd.{{ stack_name }}
 [nodes:children]
 masters
 infra.{{ stack_name }}
+lb.{{ stack_name }}
```

Comment:
@tomassedovic if outside of the OSEv3 group, the lb group vars do not contain cluster specific vars like `openshift_deployment_type`, thus failing to deploy. If included, it gets docker installed there and other looking unnecessary things done. How shall we address that?